include/bluetooth/scan.rst

```diff
@@ -80,6 +80,35 @@ Filter modes
 |             | Otherwise, the not found callback is called.                                    |
 +-------------+---------------------------------------------------------------------------------+
 
+Directed advertising
+====================
+
+The Scanning module also supports the Directed Advertising. When scanning filters are set, the filter not match event can be used
```

Comment:
I think it should be either:
filter no match event 
or
filter not matched event

Reply:
Right